themes/custom/droopler_dorg_subtheme/config/install/block.block.languageswitcher.yml

```diff
@@ -0,0 +1,21 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - language
+  theme:
+    - droopler_dorg_subtheme
+_core:
+  default_config_hash: _c6tt6_A2RBQe3ZFuUPGgUfFh9hYd2gug1JNawdS8OI
```

Comment:
Luknij jeszcze czy w zwykłym subtheme był ten `default_config_hash`, żebyśmy nie wyeksporowali jakiegoś specyficznego ID

Reply:
no sa dwa takie same mozna to wywalic poprostu czy zamienic na jakis inny?